RHIDP-6507 - modularize _Monitoring and logging with AWS (#1089)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/assembly-monitoring-and-logging-with-aws.adoc

> [id='proc-rhdh-monitoring-logging-aws_{context}']

Author: ```suggestion
[id="assembly-monitoring-and-logging-with-aws_{context}"]
```

> = Monitoring and logging with Amazon Web Services (AWS) in {product}

Author: ```suggestion
= Monitoring and logging {product} on {aws-brand-name} ({aws-short})
```

You can configure {product} to use Amazon CloudWatch for real-time monitoring and Amazon Prometheus for comprehensive logging.

> This is convenient when {product-short} is hosted on {aws-brand-name} ({aws-short}) infrastructure.

Author: ```suggestion
This is convenient when hosting {product-short} on {aws-brand-name} ({aws-short}) infrastructure.
```

include::assembly-monitoring-with-amazon-prometheus.adoc[leveloffset=+1]

> include::assembly-logging-with-amazon-cloudwatch-logs.adoc[leveloffset=+1]

Author: ```suggestion
include::assembly-logging-with-amazon-cloudwatch.adoc[leveloffset=+1]
```

[id='proc-rhdh-monitoring-logging-aws_{context}']
= Monitoring and logging with Amazon Web Services (AWS) in {product}

> In the {product}, monitoring and logging are facilitated through Amazon Web Services (AWS) integration. With features like Amazon CloudWatch for real-time monitoring and Amazon Prometheus for comprehensive logging, you can ensure the reliability, scalability, and compliance of your {product-short} application hosted on AWS infrastructure.

Reviewer: We need to be mindful of our language for legal reasons and be careful not to promise anything in prod docs 
```suggestion
In the {product}, monitoring and logging are facilitated through Amazon Web Services (AWS) integration. With features like Amazon CloudWatch for real-time monitoring and Amazon Prometheus for comprehensive logging, you can help meet the reliability, scalability, and compliance needs of your {product-short} application hosted on AWS infrastructure.
```

> This integration enables you to oversee, diagnose, and refine your applications in the Red Hat ecosystem, leading to an improved development and operational journey.

Reviewer: CCS strives to use more customer-centric language and avoid product-centric language. In general, "enables you" or "allows you" tends to be product-centric. To make this more user-centric, focus on what the user can do with the product vs. what the product can do.

> This integration enables you to oversee, diagnose, and refine your applications in the Red Hat ecosystem, leading to an improved development and operational journey.

Reviewer: Again, be careful not to "promise" certain outcomes in the prod docs.

> In the {product}, monitoring and logging are facilitated through Amazon Web Services (AWS) integration. With features like Amazon CloudWatch for real-time monitoring and Amazon Prometheus for comprehensive logging, you can help meet the reliability, scalability, and compliance needs of your {product-short} application hosted on AWS infrastructure.
> 
> This integration enables you to oversee, diagnose, and refine your applications in the Red Hat ecosystem, leading to an improved development and operational journey.

Author: ```suggestion
You can configure {product} to use Amazon CloudWatch for real-time monitoring and Amazon Prometheus for comprehensive logging.
This is convenient when {product-short} is hosted on {aws-brand-name} ({aws-short}) infrastructure.
```
